helpers/constants.js

```diff
@@ -1,28 +1,28 @@
 exports.constants = {
-	admin: {
-		name: "admin",
-		email: "a0k03lf@gmail.com"
-	},
-	confirmEmails: {
-		from: "a0k03lf@gmail.com"
-	},
-	userTypes: {
-		RECRUITER: "RECRUITER",
-		JOBSEEKER: "JOBSEEKER"
-	},
-	experienceType: {
-		FRESHER: 'FRESHER',
-		PROFESSIONAL: 'PROFESSIONAL'
-	},
-	expertiseLevel: {
-		FRESHER: "FRESHER",
-		INTERMIDIATE: "INTERMIDIATE",
-		EXPERIENCED: "EXPERIENCED"
-	},
-	applicationStatus: {
-		APPLIED: "APPLIED",
-		INTERVIEWING: "INTERVIEWING",
-		HIRING: "HIRING",
-		REJECTED: "REJECTED"
-	}
+  admin: {
+    name: "admin",
+    email: "a0k03lf@gmail.com",
+  },
+  confirmEmails: {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 1 tab but found 2 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7787263)
```suggestion
	confirmEmails: {
```